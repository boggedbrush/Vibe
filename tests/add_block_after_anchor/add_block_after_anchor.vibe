# VibeSpec: 1.2
patch_type: add_block
file: hello.py
position: after
anchor: "^def greet"
--- code: |
  # >>> Inserted after the greet() function
  greet = wrap_with_logging(greet)
