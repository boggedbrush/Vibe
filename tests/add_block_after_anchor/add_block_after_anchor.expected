# hello.py
def greet(name):
    print(f"Hello, {name}!")


# >>> Inserted after the greet() function
greet = wrap_with_logging(greet)


class Greeter:
    def __init__(self, name):
        print(f'Hello {name}')
